src/modules/partition/gui/EncryptWidget.cpp

```diff
@@ -201,3 +213,13 @@ EncryptWidget::onCheckBoxStateChanged( int checked )
 
     updateState();
 }
+
+void
+EncryptWidget::setFilesystem( const QString& fs )
+{
+    m_filesystem = fs;
+    if ( m_state != Encryption::Disabled )
+    {
+        updateState( false );
```

Comment:
I don't get why you would want to avoid an update-notification here?

Reply:
Changes to the encryptwidget trigger a call to `ChoicePage::onActionChanged`.  That in turn calls `EncryptWidget::setFilesystem` which was then triggering a signal storm.